components/03-modules/slider/_slider.scss

```diff
@@ -0,0 +1,186 @@
+$slider__margin                  : $spacer--medium !default;
+$slider__transition              : $transition-base !default;
+$slider__title-margin            : $spacer--large !default;
+$slider__navigation-margin       : $spacer !default;
+$slider__navigation-icon-spacing : 48px !default;
+$slider__navigation-icon-margin  : $spacer !default;
+$slider__name-margin-right       : $spacer !default;
+$slider__name-font-weight        : $font-weight-base !default;
+$slider__name-font-size          : $font-size-base !default;
+$slider__item-padding-right      : $spacer--medium !default;
+$slider__image-margin            : 0 auto $spacer auto !default;
+$slider__badge-left              : $spacer !default;
+
+$slider__price-old-margin        : $spacer !default;
+$slider__price-font-weight       : $font-weight-bold !default;
+$slider__price-color             : $black !default;
+$slider__price-old-color         : $black !default;
```

Comment:
Is there really a need for doubling this ?

Reply:
exterminated, thanks